pkg/phlaredb/head.go

```diff
@@ -1003,6 +1004,7 @@ func (h *Head) flush(ctx context.Context) error {
 	h.meta.Files = files
 	h.meta.Stats.NumProfiles = uint64(h.profiles.index.totalProfiles.Load())
 	h.meta.Stats.NumSamples = h.totalSamples.Load()
+	h.meta.Compaction.Sources = []ulid.ULID{h.meta.ULID}
```

Comment:
Original block should have their own sources this is how the Prometheus format expect it.

Reply:
cc @pracucci if we don't have local compaction that is correct right ?

Reply:
Confirmed this with Marco and reading more Prometheus.